src/ray/object_manager/plasma/test/allocator_test.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost/filesystem.hpp>
+#include "gtest/gtest.h"
+#include "ray/object_manager/plasma/plasma_allocator.h"
+
+using namespace boost::filesystem;
+
+namespace plasma {
+namespace {
+const int64_t kMB = 1024 * 1024;
+std::string CreateTestDir() {
+  path directory = temp_directory_path() / unique_path();
+  create_directories(directory);
+  return directory.native();
+}
+};  // namespace
+
+TEST(PlasmaAllocatorTest, NoFallbackPassThroughTest) {
+  auto plasma_directory = CreateTestDir();
+  auto fallback_directory = CreateTestDir();
+  int64_t kLimit = 256 * sizeof(size_t) + 10 * kMB;
```

Comment:
Btw can we make `256 * sizeof(size_t)` some constant number?

Reply:
for sure! we should have a linter for no magic number.

Reply:
i think those magic number in tests are probably fine as I don't want to leak it to caller. but added kDlMallocReserved